perfmonitor.go

```diff
@@ -10,11 +10,11 @@ import (
 	"net/http"
 	"os"
 
-	"./perfstats"
+	"github.com/Seneca-CDOT/StatUtils/perfstats"
 )
 
 // AppVersion - current app version
-const AppVersion = "1.1.0"
+const AppVersion = "0.0.1"
```

Comment:
What style of versioning are you using? One option is going the [semantic](https://semver.org/) route.

Reply:
@poftadeh yeah, this is what I was looking at